discourse/discourse.dockerfile

```diff
@@ -44,7 +44,9 @@ RUN sudo mkdir /var/run/postgresql/9.5-main.pg_stat_tmp && sudo chown postgres:p
 WORKDIR /home/user/discourse
 
 # Configure Cloud9 to use Discourse's source directory as workspace (-w).
-RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/discourse/" /etc/supervisord.conf
+RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/discourse/" /etc/supervisord.conf && \
+    echo "export DISCOURSE_PORT=\"3000\"" >> /home/user/.bashrc && \
```

Comment:
Nit: At some point we switched from `echo` to `printf`, because it has a more stable behavior across different shells (especially with `\n` characters).